components/reviews-card/index.tsx

```diff
@@ -0,0 +1,33 @@
+import Image, { StaticImageData } from "next/image";
+import React from "react";
+import Ellipse from "../../assets/svg/ellipse.svg";
```

Comment:
lets not use file paths like this, if I am not mistaken, we can import from root as follows: 
`@/assets/svg/ellipse.svg`, makes imports cleaner